### ISSUE TITLE
Cannot Enable Mouse Smoothing For Mouse Region

### ISSUE THREAD

**Issue by GlacialTurtle** [kozec/sc-controller#172]
Pretty much as the title - mouse smoothing seemingly can't be applied for mouse region, even when putting it in the custom action editor (doesn't show in bottom display, no syntax warning).

**Comment by kozec** [comment]
Yeah, that's pretty-much by design. Mouse region simply takes position on input and translates it to position on screen. Deadzone is probably possible, but feedback and smoothing computes with changes in positions, what doesn't have any meaning with it.

**Comment by LinuxJoeUser** [comment]
Looking back at this, I'm wondering why feedback and smoothing are disabled for M. Region?  They're available for mouse/trackball on the full-size screen, why not for mouse region?  It seems like mouse region would just be an artificial screen size as far as the software is concerned.  

Certainly of the two, being able to smooth the apparent epileptic seizure that occurs with Mouse Region would be the most useful.
